zeppelin-interpreter/src/main/java/org/apache/zeppelin/dep/DependencyResolver.java

```diff
@@ -113,6 +113,28 @@ public DependencyResolver(String localRepoPath) {
     return libs;
   }
 
+  public List<File> load(String artifact, File destPath) throws IOException, RepositoryException {
```

Comment:
Seems like this method with same signature already exists [above]( https://github.com/Leemoonsoo/zeppelin/blob/755912c72458326a25bb26f9bf249a4a4bbf3b1b/zeppelin-interpreter/src/main/java/org/apache/zeppelin/dep/DependencyResolver.java#L66).